### DISCUSSION TITLE
Add choice of tab unloader

### DISCUSSION THREAD

**Discussion by tranquil-tr0** [zen-browser/desktop#3126] [Ideas]
Suggestion to expose option between native Firefox tab unloader and Zen Browser tab unloader. This could also be a question, since I can't quite understand why the Zen Browser tab unloader would be the default (maybe to allow for the right-click context menu to show unload option?), so also a question. For context, the option I am referring to can be found in `about: config` under `browser.tabs.unloadOnLowMemory` and `zen.tab-unloader.enabled`. My personal preference comes from the default being responsive to memory use, which I would imagine could benefit others, as evidenced by [#3076](https://github.com/zen-browser/desktop/discussions/3076).

**Comment by mokazemi** [comment]
I'm having the same suggestion too as a daily user:

**Add support for different options and ways of tab unloader (unload after a period of inactivity, unload on low memory, unload manually, etc.)**

**Reply by tranquil-tr0** [reply]
This would make a lot more sense. Since I assume there is some advantage to using Zen tab unloader, if it is approachable the superior solution is clearly to integrate "on low memory" features, etc.

**Comment by ABruel** [comment]
I need this. Firefox's default tab unloader is a mess. Eats my RAM unnecessarily by keeping a bunch of old tabs open just because I have a lot of free RAM. I don't want my browser thinking it has a right to all of my RAM.

**Comment by tranquil-tr0** [comment]
This is no longer relevant, as v1.12.9b changed the tab discarder/unloader method to use the Firefox default.
